py/tests/release.py

```diff
@@ -72,7 +78,9 @@
             "config": '''[
 {aecore,
  [
-  {peers, []}
+  {peers, []},
+  {aec_pow_cuckoo, {"lean16", "-t 5", 16}},
```

Comment:
These PoW-related settings will conflict with PR #454 - or PR #454 will conflict with this (whatever is ready for merge last).